README.md

```diff
@@ -1,4 +1,4 @@
-# gulp-eslint [![Build Status](https://travis-ci.org/adametry/gulp-eslint.svg)](https://travis-ci.org/adametry/gulp-eslint) [![Coverage Status](https://img.shields.io/coveralls/adametry/gulp-eslint.svg)](https://coveralls.io/r/adametry/gulp-eslint)
```

Comment:
No need to change this. This badge is currently not broken.

According to my own experience the official coverall badge seems to be cached too long and doesn't reflect the latest status immediately.